data-sources/flutter-android/config.yml

```diff
@@ -0,0 +1,15 @@
+id: flutter-android
+displayName: Flutter Android
+description: |
+  Instrument and monitor your Flutter Android app with New Relic’s Android and Flutter agent. Analyze your performance data (Errors, Application Crashes, App Launches and App Installations) to diagnose, improve, and optimize your code.
```

Comment:
@rahulbasu-nr , need your expertise

Reply:
I have updated it as per your comments.
Thank you...